### PR TITLE
format: remove bash from exceptions and format it

### PR DESCRIPTION
**Pull request by jian-lin**
### Description
There is no open pr against bash now.

### Checklist

- [X] Change is backwards compatible.

- [X] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [X] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: modules/programs/bash.nix [hunk 1]
```diff
@@ -175,80 +169,71 @@ in
...
-        ));
-    in mkIf cfg.enable {
-      home.file.".bash_profile".source = writeBashScript "bash_profile" ''
-        # include .profile if it exists
-        [[ -f ~/.profile ]] && . ~/.profile
-
-        # include .bashrc if it exists
-        [[ -f ~/.bashrc ]] && . ~/.bashrc
-      '';
+  config = (let
```
Comment (ncfavier) [inline modules/programs/bash.nix:172]:
```suggestion
  config = let
```

While you're at it

**Review by ncfavier — APPROVED** [review]

**Comment by rycee** [comment]
Thanks, that's great! Rebased to master in bb4b25b302dbf0f527f190461b080b5262871756.

**Comment by ncfavier** [comment]
(Meta: it's good to be able to easily keep track of which PRs were merged and which were closed. We should use the merge button whenever possible; maybe enable the "rebase and merge" option?)

**Comment by sumnerevans** [comment]
Rebase and merge would cause PRs with multiple commits to be merged into master as multiple commits. Maintainers would have to be sure to check the commit history to make sure that all of the commits are good (which I've been known to forget to do in the past).

**Comment by ncfavier** [comment]
It seems like there's an option to update branches (by rebasing) without merging: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/keeping-your-pull-request-in-sync-with-the-base-branch

We could enable that and stick to squash merges.

**Pull request closed without merging**